packages/common/src/permissions/_internal/checkAssertion.ts

```diff
@@ -298,8 +320,18 @@ function arrayAssertions(
         }
       }
     }
-    if (assertion.type === "without" && arrayContainsValue) {
-      response = "array-contains-invalid-value";
+    if (assertion.type === "without") {
+      if (Array.isArray(val)) {
+        const containsNone = val.every((v) => !arrayProp.includes(v));
+        if (!containsNone) {
```

Comment:
Super minor -- could we update this to something like 
```ts
containsSome = val.some((v) => arrayProp.includes(v));
if (containsSome) {
 response = "array-contains-invalid-value";
}
```
just to remove all of the negation? May make it easier to grok -- and I _think_ that's the same logic as written? But double check me on that...😅

Reply:
Good idea!